### DISCUSSION TITLE
Improve doc/UI for the health check feature

### DISCUSSION THREAD

**Discussion by isoppp** [coollabsio/coolify#1836] [Feature Requests]
Thank you for the great product!

I have been trying coolify but encounted an issue that I could not achieve zero-downtime deployments using Dockerfile which has some scripts before running the server.

After investigation, it seems like the feature is not implemented yet, as expected from UI (read [this function](https://github.com/coollabsio/coolify/blob/46ed17c99e2b0b6aae51d6490aa5c91b59b7ff08/app/Jobs/ApplicationDeploymentJob.php#L773-L815)).

However, the code seems to check docker instance's healthiness. So I modified Dockerfile to utilize [HEALTHCHECK](https://docs.docker.com/reference/dockerfile/#healthcheck) and successfully achieved zero-downtime deployments.

So it would be good to improve documentation or hide unused UI to prevent misunderstanding.

Sorry if it's not appropriate place for the doc improvement or I misunderstood the code.

**Comment by ershisan99** [comment]
@isoppp do you mind sharing a bit more about how exactly you managed to achieve zero-downtime deployments? I'm looking into it myself right now and your experience might help me out a bit. Cheers.

**Reply by isoppp** [reply]
@ershisan99 
(I'm not currently using Coolify so my information might be outdated.)

As far as I remember I tested with this Dockerfile and it worked successfully in that time.
https://github.com/isoppp/paas-verification-tiny-nodejs/blob/dockerize-healthcheck/Dockerfile#L13-L14

But after this post, I think coolify had added their own health check feature to their app.


If you need more information, please ping me in Feb. I'll have more time to look into this again in detail.
